src/Service.Tests/Unittests/RuntimeConfigLoaderJsonDeserializerTests.cs

```diff
@@ -301,8 +413,7 @@ public static string GetModifiedJsonString(string[] reps, string enumString)
   },
   ""data-source"": {
     ""database-type"": " + enumString + @",
-    ""connection-string"": ""server=dataapibuilder;database=" + reps[++index % reps.Length] + @";uid=" + reps[++index % reps.Length] + @";Password=" + reps[++index % reps.Length] + @";"",
-    ""resolver-config-file"": """ + reps[++index % reps.Length] + @"""
+    ""connection-string"": ""server=dataapibuilder;database=" + reps[++index % reps.Length] + @";uid=" + reps[++index % reps.Length] + @";Password=" + reps[++index % reps.Length] + @";""
```

Comment:
if we would have modified this string to include `options` property with an environment variable, would it have been sufficient for testing?

Reply:
it would have been, but since there are so many different cases with data-source, i didn't wanted to merge it with it.